samples/cpp/tutorial_code/Histograms_Matching/EqualizeHist_Demo.cpp

```diff
@@ -24,10 +24,10 @@ int main( int, char** argv )
   const char* equalized_window = "Equalized Image";
 
   /// Load image
-  src = imread( argv[1], 1 );
+  src = imread( argv[1] );
```

Comment:
I would change this to `imread( argv[1] , IMREAD_COLOR);`. Because this is a tutorial, it is better to give all possible parameters to a function, than to simply rely on the default value.